config/build.ts

```diff
@@ -464,23 +458,55 @@ async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProce
         */
         const groupNumberRegex = /^(\d+\s)?(\w.*)$/;
 
-        items.forEach(item => {
+        let modifiedItems = cloneDeep(items);
+        modifiedItems.forEach(item => {
             item.group = item.group.replace(groupNumberRegex, '$2');
 
-            // Also remove "order", it's no longer needed (the snippets themselves are already in an ordered array in the YAML file)
+            // Also remove "order", "host" and "type", they are no longer needed (the snippets themselves are already in an ordered array in the YAML file, which is itself tied to host)
             delete item.order;
+            delete item.host;
+            delete item.type;
         });
 
-        let contents = jsyaml.safeDump(items, {
+        let contents = jsyaml.safeDump(modifiedItems, {
             skipInvalid: true /* skip "undefined" (e.g., for "order" on some of the snippets) */
         });
 
-        await writeFile(path.resolve(`${dir}/${host}.yaml`), contents);
+        /* Group private samples with public samples in one playlist */
```

Comment:
Whoa.  Shouldn't the yaml playlist only contain the public ones?  Maybe I'm just confused by the comment....